content/docs/concepts/architecture/_index.md

```diff
@@ -20,26 +21,51 @@ Autonity extends Ethereum at three logical layers:
 - Application layer: protocol smart contracts:
 	- **Autonity Protocol Contract** implementing protocol primitives for governance, tokenomics, liquid staking, and staking rewards distribution.
 	- **Liquid Newton** contracts for validator-specific liquid stake tokens.
+	- **Accountability Contract** implementing protocol primitives for accountability and fault detection of failures to adhere to consensus rules by consensus committee members during Tendermint consensus rounds.
+	- **Autonity Oracle Contract** implementing protocol primitives for computing median price data and managing the set of currency pairs for which Autonity's oracle network provides price data.
+	
+	Protocol smart contracts are part of the client binary. _Liquid Newton_ smart contracts are deployed on validator registration.
 
-	Autonity Protocol smart contracts are part of the client binary. _Liquid newton_ smart contracts are deployed on validator registration.
+- Consensus layer: blockchain consensus provided by the **Proof of Stake Tendermint BFT** protocol. Blocks are proposed by validators and selected by the committee for inclusion in the blockchain, with finality. The consensus mechanism enables dynamic consensus committee selection using a stake-weighting algorithm, maximising the amount of stake securing the system. Consensus is further extended to:
+  - provide accountability and fault detection for failure to adhere to consensus rules by committee members, implementing slashing penalties and a Penalty Absorbing Stake (PAS) model.
```

Comment:
I feel like these two points about accountability and oracle do not belong under the Consensus layer paragraph. They are both decoupled from consensus. 

It is true that validators have to run both the FD and the oracle as part of their "validator duties", but they are not part of the consensus process.

Reply:
Clear, thanks! Removed and edited in commit f0abfb26c62401bfb3d3de49897a639a5944840c.

Reply:
Does the edit resolve the problem, please, @lorenzo-dev1 ?